src/test/common/steps/enter-name-steps.js

```diff
@@ -54,6 +54,16 @@ Then('I am informed that the first name is too long', async function () {
   expect(errorMessage).to.be.equal('Enter a shorter first or given name')
 })
 
+Then(/^I see the first name I entered in the textbox$/, async function () {
```

Comment:
As this is specifically checking for the invalid long name, I'd prefer this step to be something like `I see the invalid first name entered in the textbox`, otherwise I'd expect it to be a valid name, or at least parameterised. Same with the method below too.